core/vulkan/cc/include/vulkan/vulkan.h

```diff
@@ -3689,6 +3690,30 @@ VKAPI_ATTR VkResult VKAPI_CALL vkCreateAndroidSurfaceKHR(
 #endif
 #endif /* VK_USE_PLATFORM_ANDROID_KHR */
 
+#define VK_USE_PLATFORM_GGP 1
+#ifdef VK_USE_PLATFORM_GGP
+
+#define VK_GGP_stream_descriptor_surface 1
```

Comment:
Should this be all capital letters?

Reply:
It looks like it should, but non-capital is the same as the rest of the equivalent #defines in this file